actions/trains.js

```diff
@@ -0,0 +1,9 @@
+module.exports = {
+  init: () => ({
+    'Red': [],
+    'Orange': [],
+    'Green': [],
+    'Blue': []
+  }),
+  setData: (store, data, actions) => data
+}
```

Comment:
this was `update.js`, but simplified (immediate return)

Reply:
(the parens are required to make the thing init returns an object (and not the start of a block)